src/dfx/src/lib/nns/install_nns.rs

```diff
@@ -580,7 +579,11 @@ pub fn upload_nns_sns_wasms_canister_wasms(env: &dyn Environment) -> anyhow::Res
                     Err(anyhow!(
                         "Failed to upload {} from {} to the nns-sns-wasm canister:\n{:?} {:?}\nStdout:\n{:?}\n\nStderr:\n{:?}",
                         upload_name,
-                        wasm_path.to_string_lossy(), command.get_program(), command.get_args(), output.stdout, output.stderr
+                        wasm_path.to_string_lossy(),
+                        command.get_program(),
+                        command.get_args(),
+                        String::from_utf8_lossy(&output.stdout),
+                        String::from_utf8_lossy(&output.stderr)
```

Comment:
These were printing stdout and stderr as vectors of integers - not very readable!